resources/ory/values.yaml

```diff
@@ -152,6 +152,7 @@ oathkeeper:
           config:
             jwks_urls:
             - http://dex-service.kyma-system.svc.cluster.local:5556/keys
```

Comment:
Is dex still required?

Reply:
I thought that I would leave it in Ory for now - for the sake of old Kyma versions, where Dex is deployed